client/webpack.config.js

```diff
@@ -0,0 +1,99 @@
+var path = require('path');
+
+var webpack           = require('webpack');
+
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var CopyPlugin        = require('copy-webpack-plugin');
+var AssetsPlugin      = require('assets-webpack-plugin');
+var BrowserSyncPlugin = require('browser-sync-webpack-plugin');
+
+var production = process.env.NODE_ENV === 'production';
```

Comment:
As far as I know, Webpack doesn't offer a flag to enable a specific mode (only a flag to explicitely activate optimizations) and rely on `NODE_ENV=production` to atomatically turn optimnizations on.

This one is only use at build time (see npm script `build:client`), but we can imagine use another ENV var to enable those parts if we want. What do you suggest to use?